lambda-search/src/lambda.rs

```diff
@@ -0,0 +1,109 @@
+mod embed;
+mod highlight;
+mod qdrant_search;
+
+use embed::Embedding;
+use futures::try_join;
+use lambda_http::{run, service_fn, Body, Error, Request, RequestExt, Response};
+use qdrant_client::client::QdrantClient;
+use qdrant_search::{
+    establish_connection, qdrant_search_embeddings, query_qdrant_text, SEARCH_LIMIT,
+};
+use reqwest::Client;
+
+// create an error response
```

Comment:
Spotted another one!

Reply:
That's a private function. One usually only documents public ones. But I'll admit that the comment isn't too helpful, so it's perhaps best to remove it.